L1Trigger/DTTriggerPhase2/test/primitivesPhase2Prod_over_hlt_cfg.py

```diff
@@ -0,0 +1,191 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("L1DTTrigPhase2Prod")
+
+process.load("Geometry.CMSCommonData.cmsIdealGeometryXML_cff")
+process.load("Geometry.DTGeometry.dtGeometry_cfi")
+process.DTGeometryESModule.applyAlignment = False
+
+process.load("L1Trigger.DTTriggerPhase2.dtTriggerPhase2PrimitiveDigis_cfi")
+process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
+process.load("Configuration.StandardSequences.MagneticField_AutoFromDBCurrent_cff")
+#process.GlobalTag.globaltag = "90X_dataRun2_Express_v2"
+process.GlobalTag.globaltag = "80X_dataRun2_2016SeptRepro_v7"
+
+#Calibrate Digis
+process.load("L1Trigger.DTTriggerPhase2.CalibratedDigishlt_cfi")
+#process.CalibratedDigis.flat_calib = 0 #turn to 0 to use the DB  , 325 for JM and Jorge benchmark
+
+#DTTriggerPhase2
+process.load("L1Trigger.DTTriggerPhase2.dtTriggerPhase2PrimitiveDigishlt_cfi")
+process.dtTriggerPhase2PrimitiveDigis.pinta = True
+
+process.source = cms.Source("PoolSource",
+                            fileNames = cms.untracked.vstring(
+        'file:/eos/cms/store/data/Run2017A/RPCMonitor/RAW/v1/000/295/655/00000/00420561-F145-E711-9694-02163E01A23B.root',
```

Comment:
in general, preferable to access eos files over xrootd rather than the fuse mount

Reply:
we will remove this file